src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.serviceimpl.DataConverterImpl;
```

Comment:
The package for implementations of services should be 'service.impl', but here it is 'service.serviceimpl'. The package naming should follow the convention provided in the checklist.